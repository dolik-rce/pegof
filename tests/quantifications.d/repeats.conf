input quantifications.d/repeats.peg
terminal-inline-limit 0
inline-limit 0
optimize
