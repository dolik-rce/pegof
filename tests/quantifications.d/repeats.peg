Z <- A1 B1 C1 A2 B2 C2 A3 B3 C3

X <- "X"

A1 <- X X

B1 <- X+ X

C1 <- X* X

A2 <- X X+

B2 <- X+ X+

C2 <- X* X+

A3 <- X X*

B3 <- X+ X*

C3 <- X* X*
