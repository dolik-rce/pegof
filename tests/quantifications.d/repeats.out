Z <- A1 B1 C1 A2 B2 C2 A3 B3 C3

X <- "X"

A1 <- X X

B1 <- X+ X

C1 <- X+

A2 <- X X+

B2 <- X+ X+

C2 <- X+

A3 <- X+

B3 <- X+

C3 <- X*
